priv/repo/migrations/20240822072422_migrate_resources5.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Animina.Repo.Migrations.MigrateResources5 do
+  @moduledoc """
+  Updates resources based on their most recent snapshots.
+
+  This file was autogenerated with `mix ash_postgres.generate_migrations`
+  """
+
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      add :registration_completed_at, :utc_datetime_usec
+    end
+
+    execute """
+    UPDATE users
+    SET registration_completed_at = date_trunc('day', now()) + interval '9 hours';
```

Comment:
Please set it to a specific timestamp and not some sort of `now()`. Like today at 09:00 am. The 22th of August 2024 at 09:00 am. Otherwise we will not be able to find those entries reliable in the future.

Reply:
Okay  , I will set it up like this